Formula/czmqpp.rb

```diff
@@ -0,0 +1,81 @@
+class Czmqpp < Formula
+  desc "C++ wrapper for czmq"
+  homepage "https://github.com/zeromq/czmqpp"
+  url "https://github.com/zeromq/czmqpp.git", :tag => "v1.2.0", :revision => "a3c38eb6e3018887a8ecabc77f3029f7edb7b2d0"
+
+  head "https://github.com/zeromq/czmqpp.git"
+
+  option :universal
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "czmq"
```

Comment:
I actually tried using `ENV.cxx11` before before, but it seems to fail on this. I'm not sure it's even valid to use in this context.

```
==> Using the sandbox
==> ["CXX"] -o test test.cpp -Os -w -pipe -march=native -mmacosx-version-min=10.11 -std=c++11 -I/usr
Error: czmqpp: failed
no implicit conversion of Array into String
```